src/compressor/snappy/SnappyCompressor.h

```diff
@@ -97,7 +97,11 @@ class SnappyCompressor : public Compressor {
     if (qat_enabled)
       return qat_accel.decompress(p, compressed_len, dst, compressor_message);
 #endif
+#if SNAPPY_VERSION <= 0x10108
     snappy::uint32 res_len = 0;
+#else
+    snappy::uint32_t res_len = 0;
+#endif
```

Comment:
this is still not quite right. `uint32_t` is not defined in `snappy` naming space evn in v1.1.9

in my case, i installed snappy into `/tmp/snappy-install`, the only occurrence is `GetUncompressedLength()`.

```console
$ grep uint32_t /tmp/snappy-install/include/* -r
/tmp/snappy-install/include/snappy.h:  bool GetUncompressedLength(Source* source, uint32_t* result);
```

when it comes to v1.1.7
```console
$ grep -w uint32 /tmp/snappy-install/include/* -r
/tmp/snappy-install/include/snappy.h:  bool GetUncompressedLength(Source* source, uint32* result);
/tmp/snappy-install/include/snappy-stubs-public.h:typedef uint32_t uint32;
/tmp/snappy-install/include/snappy-stubs-public.h:typedef unsigned int uint32;
```

so i'd suggest just put
```c++
uint32_t res_len = 0;
```
unconditionally.

Reply:
OK

Reply:
@smithfarm can you drop the `#ifdef` guards, and just put
```c++
uint32_t res_len = 0;
```
unconditionally, simpler this way.